support-frontend/app/services/PayPalNvpService.scala

```diff
@@ -79,6 +80,7 @@ class PayPalNvpService(apiConfig: PayPalConfig, wsClient: WSClient) extends Touc
 
   // Sets up a payment by contacting PayPal and returns the token.
   def retrieveToken(returnUrl: String, cancelUrl: String)(billingDetails: PayPalBillingDetails): Future[Option[String]] = {
+    val noShipping = if(billingDetails.requireShippingAddress) "0" else "1"
```

Comment:
We don't want to ask the user to pick an address when using PayPal on the regular checkouts, only when using the 'one click' button. This controls that.

Reply:
lint missing space
```suggestion
    val noShipping = if (billingDetails.requireShippingAddress) "0" else "1"
```